--- pytest-flake8-0.8.1/test_flake8.py.orig	2017-04-18 20:13:18.303577852 +0200
+++ pytest-flake8-0.8.1/test_flake8.py	2017-04-18 20:14:15.383577200 +0200
@@ -128,7 +128,8 @@
         "*E201*",
         "*1 failed*",
     ])
-    assert 'passed' not in result.stdout.str()
+    # false positive in debug log for registering --stdin-display-name option
+    # assert 'passed' not in result.stdout.str()
 
 
 @pytest.mark.xfail("sys.platform == 'win32'")
